vocabs/DID-core-shape-v1.jsonld

```diff
@@ -0,0 +1,228 @@
+{
+    "@context": {
+        "path": {
+            "@id": "http://www.w3.org/ns/shacl#path",
+            "@type": "@id"
+        },
+        "pattern": {
+            "@id": "http://www.w3.org/ns/shacl#pattern"
+        },
+        "nodeKind": {
+            "@id": "http://www.w3.org/ns/shacl#nodeKind",
+            "@type": "@id"
+        },
+        "datatype": {
+            "@id": "http://www.w3.org/ns/shacl#datatype",
+            "@type": "@id"
+        },
+        "minCount": {
+            "@id": "http://www.w3.org/ns/shacl#minCount",
+            "@type": "http://www.w3.org/2001/XMLSchema#integer"
+        },
+        "targetClass": {
+            "@id": "http://www.w3.org/ns/shacl#targetClass",
+            "@type": "@id"
+        },
+        "targetObjectsOf": {
+            "@id": "http://www.w3.org/ns/shacl#targetObjectsOf",
+            "@type": "@id"
+        },
+        "xone": {
+            "@id": "http://www.w3.org/ns/shacl#xone",
+            "@type": "@id"
+        },
+        "not": {
+            "@id": "http://www.w3.org/ns/shacl#not",
+            "@type": "@id"
+        },
+        "property": {
+            "@id": "http://www.w3.org/ns/shacl#property",
+            "@type": "@id"
+        },
+        "targetSubjectsOf": {
+            "@id": "http://www.w3.org/ns/shacl#targetSubjectsOf",
+            "@type": "@id"
+        },
+        "maxCount": {
+            "@id": "http://www.w3.org/ns/shacl#maxCount",
+            "@type": "http://www.w3.org/2001/XMLSchema#integer"
+        },
+        "first": {
+            "@id": "http://www.w3.org/1999/02/22-rdf-syntax-ns#first",
+            "@type": "@id"
+        },
+        "rest": {
+            "@id": "http://www.w3.org/1999/02/22-rdf-syntax-ns#rest",
+            "@type": "@id"
+        },
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "sh": "http://www.w3.org/ns/shacl#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "did": "https://www.w3.org/ns/did/vocab/v1#"
+    },
+    "@graph": [
+        {
+            "@id": "_:b0",
+            "path": "did:controller",
+            "pattern": "^did:[a-z0-9]+:[a-zA-Z0-9.-_:]+"
+        },
+        {
+            "@id": "_:b1",
+            "nodeKind": "sh:IRI",
+            "path": "did:capabilityInvocation"
+        },
+        {
+            "@id": "_:b10",
+            "nodeKind": "sh:IRI",
+            "path": "did:capabilityDelegation"
+        },
+        {
+            "@id": "_:b11",
+            "sh:minCount": 1,
+            "nodeKind": "sh:BlankNodeOrIRI",
+            "path": "did:serviceEndpoint"
+        },
+        {
+            "@id": "_:b12",
+            "@type": "sh:NodeShape",
+            "property": [
+                "_:b10",
+                "_:b13",
+                "_:b14",
+                "_:b1",
+                "_:b9",
+                "_:b0",
+                "_:b15",
+                "_:b16",
+                "_:b17"
```

Comment:
Can we use more descriptive blank node names here? For example _:controller, _:serviceEndpoint, etc.

Reply:
Well... I write the shacl (and the vocabulary) in Turtle and I let some tool translate it into JSON-LD. Once everything is final I can try to do some manual rewrite of these names, but not until we are absolutely final.